File: SWB4/Client/OpenOffice/SWBOfficeCore/src/org/semanticwb/openoffice/UserInfo.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package org.semanticwb.openoffice;

import java.util.Map;




/**
 *
 * @author victor.lorenzana
 */
public final class UserInfo 
{
    private String password, login;
    private Map<String,String> cookies=new java.util.HashMap<String,String>();
    public UserInfo(String password,String login)
    {
        this.login=login;
        this.password=password;
    }
    public Map<String,String> getCookies()
    {        
        return cookies;
    }
    public String getLogin()
    {
        return this.login;
    }
    public void getSession()
    {
        
    }
    public String getPassword()
    {
        return password;
    }
    void changePassword(String newPassword,String oldPassword)
    {
        this.password=newPassword;
        /*if(!oldPassword.equals(password))
        {
            // muestra mensaje de error
        }
        else
        {
            // cambia el password        
        }*/
    }
}

File: SWB4/Client/Microsoft Office/INFOTEC WebBuilder 4/WBOffice4/UserInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WBOffice4
{
    internal class UserInfo
    {
        private String password, login;
        private Dictionary<string, string> cookies = new Dictionary<string, string>();        


        public UserInfo(String login,String password)
        {
            this.login = login;
            this.password = password;
        }        
        
        public String Login
        {
            get
            {
                return this.login;
            }
        }
        public String Password
        {
            get
            {
                return this.password;
            }
        }
        public Dictionary<string, string> Cookies
        {
            get
            {
                return cookies;
            }
            set
            {
                cookies = value;
            }
        }        
        public void ChangePassword(String newPassword)
        {            
            this.password = newPassword;            
        }
    }
}
